sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/clustering/utils/Common.scala

```diff
@@ -0,0 +1,71 @@
+package net.sansa_stack.ml.spark.clustering.utils
+
+import java.io.PrintWriter
+
+import org.apache.jena.graph.{ NodeFactory, Triple}
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.json4s.DefaultFormats
+import org.json4s.jackson.Serialization
+
+import net.sansa_stack.ml.spark.clustering.datatypes.{Cluster, Clusters, Poi}
+
+object Common {
+  val prefixID = "http://example.org/id/poi/"
+  val prefixCategory = "http://example.org/hasCategory"
+  val prefixCoordinate = "http://example.org/id/hasCoordinate/"
+
+
+  /**
+    * create a pair RDD and join with another pair RDD
+    *
+    * @param sparkContext
+    * @param ids an array with poi id
+    * @param pairs
+    * @return an array of poi
+    */
+  def join(sparkContext: SparkContext, ids: Array[Long], pairs: RDD[(Long, Poi)]): Array[Poi] = {
```

Comment:
Is this list `ids` big? or does it increases with the size of the dataset? If so, we have to consider having RDDs instead?